Update and extend lazer mod articles (#14794)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Classic/en.md

| :-- | :-- |
| Acronym | CL |
| Type | Conversion |
| Game modes | ![][osu!] ![][osu!taiko] ![][osu!mania] |
| Score multiplier | `0.96x` |
| Status | Unranked |
| Incompatible mods ![][osu!] | [Strict Tracking (ST)](/wiki/Gameplay/Game_modifier/Strict_Tracking) |

> | Incompatible mods ![][osu!taiko] ![][osu!mania] | None |

Reviewer: probably worth adding osu!catch here as well, and to the `Game modes` row, too?

Author: I didn't add it because Classic isn't a selectable mod in osu!catch, it's merely applied to stable scores. (As far as I'm aware there is no difference between stable and lazer gameplay in catch.)

Reviewer: ah, alright

:::

The **Classic** mod emulates gameplay mechanics from osu!(stable) that have since changed.

<!-- TODO probably more detail above -->

When viewed from osu!(lazer) or with `Lazer mode` enabled on the website, scores set in osu!(stable) automatically have this mod enabled with its default settings. The mod's unranked status does not affect the scores in this case.

## Customisation

![Classic customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/CL.png)

- **No slider head accuracy requirement** (default: enabled): [Sliders](/wiki/Gameplay/Hit_object/Slider) will work the same way they used to in osu!(stable), meaning slider heads do not have to be hit accurately.
- **Apply classic note lock** (default: enabled): [Notelock](/wiki/Gameplay/Judgement/Notelock) will work the same way it used to in osu!(stable).
- **Always play a slider's tail sample** (default: enabled): A slider's tail sample will always be played, even if it was not hit.
- **Fade out hit circles earlier** (default: enabled): Hit circles will fade out into a miss, rather than after it.

> - **Classic health** (default: enabled): [Health drain](/wiki/Gameplay/Health) will work similar to how it used to in osu!(stable).

Reviewer: ```suggestion
- **No slider head accuracy requirement** (default: enabled): [Sliders](/wiki/Gameplay/Hit_object/Slider) will work the same way as in osu!(stable), meaning slider heads do not have to be hit accurately.
- **Apply classic note lock** (default: enabled): [Notelock](/wiki/Gameplay/Judgement/Notelock) will work the same way as in osu!(stable).
- **Always play a slider's tail sample** (default: enabled): A slider's tail sample will always be played, even if it was not hit.
- **Fade out hit circles earlier** (default: enabled): Hit circles will fade out into a miss, rather than after it.
- **Classic health** (default: enabled): [Health drain](/wiki/Gameplay/Health) will work similarly to osu!(stable).
```

Reviewer: "Not dead yet!"

> *Note that these settings are only available in osu!. In osu!taiko and osu!mania, this mod cannot be customised.*

Reviewer: ```suggestion
The above settings are only available in the osu! game mode.
```
no need to mention other modes imo if it's only applicable for one mode

<!-- TODO probably more detail above -->

> When viewed from osu!(lazer) or `Lazer mode` on the website, scores set in osu!stable automatically have this mod enabled with its default settings. The mod's unranked status does not affect the scores in this case.

Reviewer 2: ```suggestion
When viewed from osu!(lazer) or with `Lazer mode` enabled on the website, scores set in osu!(stable) automatically have this mod enabled with its default settings. The mod's unranked status does not affect the scores in this case.
```

## Customisation

![Classic customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/CL.png)

> - **No slider head accuracy requirement** (default: enabled): [Sliders](/wiki/Gameplay/Hit_object/Slider) will work the same way they used to in osu!stable, so slider heads don't have to be hit accurately.

Reviewer 2: ```suggestion
- **No slider head accuracy requirement** (default: enabled): [Sliders](/wiki/Gameplay/Hit_object/Slider) will work the same way they used to in osu!(stable), meaning slider heads do not have to be hit accurately.
```

> - **Apply classic note lock** (default: enabled): [Notelock](/wiki/Gameplay/Judgement/Notelock) will work the same way it used to in osu!stable.

Reviewer 2: ```suggestion
- **Apply classic note lock** (default: enabled): [Notelock](/wiki/Gameplay/Judgement/Notelock) will work the same way it used to in osu!(stable).
```

> - **Always play a slider's tail sample** (default: enabled): A slider's tail sample will always be played, even if it wasn't hit.

Reviewer 2: ```suggestion
- **Always play a slider's tail sample** (default: enabled): A slider's tail sample will always be played, even if it was not hit.
```

- **Fade out hit circles earlier** (default: enabled): Hit circles will fade out into a miss, rather than after it.

> - **Classic health** (default: enabled): [Health](/wiki/Gameplay/Health) will work similar to how it used to in osu!stable.

Reviewer 2: ```suggestion
- **Classic health** (default: enabled): [Health drain](/wiki/Gameplay/Health) will work similar to how it used to in osu!(stable).
```
